nemo/collections/asr/parts/utils/decoder_timestamps_utils.py

```diff
@@ -670,7 +665,7 @@ def run_ASR_BPE_CTC(self, asr_model: Type[EncDecCTCModelBPE]) -> Tuple[Dict, Dic
                     hyp_words, word_ts = self.run_pyctcdecode(log_prob, onset_delay_in_sec=onset_delay_in_sec)
                 else:
                     logits_len = torch.from_numpy(np.array([len(greedy_predictions_list)]))
-                    greedy_predictions_list = greedy_predictions_list[onset_delay:-mid_delay]
+                    greedy_predictions_list = greedy_predictions_list[onset_delay:]
```

Comment:
can you elaborate why this change?

Reply:
This was fixed because of (this issue)[https://github.com/NVIDIA/NeMo/issues/3878]. Sorry that I missed this update while commenting.